### DISCUSSION TITLE
Styling the headers at the top of the email

### DISCUSSION THREAD

**Discussion by krangor** [buttondown/discussions#41] [Q&A] [answered]
Hi!

Been wondering how to style the header line of the email itself (not the ones in the body of the text), as these don't seem to be affected by the rest of the CSS stylesheet (unless I'm wrong!) Would love to be able to choose a serif font, center it, etc etc. Thanks!

**Comment by jmduke** [comment]
@krangor Are you referring to in the email or in the web archive? (We've got different answers for both!)

**Comment by krangor** [comment]
Specifically the email!

—
Mark Slutsky
***@***.***
www.markslutsky.com
—

**Reply by jmduke** [reply, marked as answer]
Both should be accessible via `h1.subject` (that is, a tag of "h1" and a class of "subject"), like so:

```
h1.subject {
  color: red;
}
```

**Comment by krangor** [comment]
Excellent! And that goes in the regular stylesheet section?

—
Mark Slutsky
***@***.***
www.markslutsky.com
—

**Reply by jmduke** [reply]
Yup!

**Comment by krangor** [comment]
Sorry to bring this up again, but I'm having a bit of trouble making this work. 
I put this code in as a test:
`h1.subject {
  color: red;
  text-align: center;
}`
into the CSS bin in the email design section, but it doesn't seem to be affecting the preview at all. Any idea what I might be doing wrong here?

(And as a side note, all the "blockquote" text in both my email AND web previews is suddenly very large and I'm confused because I hadn't made any changes.)

**Reply by jmduke** [reply]
Ah, this is a bit of a convoluted answer:

1. You've switched over to use the `classic` email template, which doesn't add the subject in by default..
2. _but_ you added the subject in the 'header' template for emails (which is fine!)
3. _but_ that doesn't have the `subject` class, since you're manually inserting it.

**Reply by krangor** [reply]
Aha. So it would have worked if I had manually put it in the body of the email, but anything in the header can't see the subject class?

**Reply by jmduke** [reply]
Less that and more — the `subject` class doesn't exist since you inserted the heading manually. (Conversely, though, if you change `h1.subject` to just `h1` it should work!)
